README.md

```diff
@@ -282,7 +282,12 @@ When using Input plugins, please be aware that input values get cached between s
       cache_inputs: false
 ```
 
- Default for backend cache is enabled
+ Backend cache is enabled by default,if you want to disable backend cache you can do:
```

Comment:
Let's add a section header to clarify this is different than the input cache.

```
### Chef InSpec Backend Cache

Chef InSpec uses a cache when executing commands and accessing files on the remote target. The cache is enabled by default. To disable the cache:
```

Reply:
Okay, thanks